### DISCUSSION TITLE
Trying to replace the menu banner

### DISCUSSION THREAD

**Discussion by oliviathevampire** [oraxen/oraxen#976] [Q&A]
Is it known that Oraxen overrides the menu banner texture? I've tried to replace it multiple times but it always ends up getting overriden again by Oraxen, any way to fix this?

**Comment by Boy0000** [comment]
just make a new glyph

**Reply by oliviathevampire** [reply]
Ah ok! What about when trying to replace the texture for the oraxen inventory?
